electrum/lnpeer.py

```diff
@@ -1462,7 +1461,7 @@ def send_node_announcement(self, alias:str):
             addrlen=len(addresses),
             addresses=addresses)
         h = sha256d(raw_msg[64+2:])
-        signature = ecc.ECPrivkey(self.privkey).ecdsa_sign(h, sigencode=ecdsa_sig64_from_r_and_s)
+        signature = ECPrivkey(self.privkey).ecdsa_sign(h, sigencode=ecdsa_sig64_from_r_and_s)
```

Comment:
This kind of cosmetic change can be committed to master independently of this pull request. It is not useful to include it in the pull request.

Reply:
This was done to be consistent with `ECPubkey`, which was already imported and used without `ecc.`